sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -1,17 +1,13 @@
 # Release History
 
-## 2.0.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (2023-04-04)
 
 ### Features Added
 - Added support for a new communication identifier `MicrosoftBotIdentifier`.
 
 ### Breaking Changes
 - Introduction of `MicrosoftBotIdentifier` is a breaking change. It will affect code that relied on using `UnknownIdentifier` with a rawID starting with `28:`
```

Comment:
I talked to the folks from iOS and they said they did not consider this behavioral breaking change to warrant a major version update, just like Python is not. I am also curious why there was a difference between those languages and C# regarding this decision. What did the Java architects say about it?

Reply:
Yes, we had a discussion around that today for iOS. For Java[(API view)](https://apiview.dev/Assemblies/Review/d70b841f485d4854870aa13688b23d2e?diffRevisionId=51937aef84e74c89b6bc685432544205&diffOnly=False&revisionId=527402c5099b4112b0559349a22548e3&doc=False) and android[(API view)](https://apiview.dev/Assemblies/Review/c8145aaf77504830b9faf0809da1018f?diffRevisionId=1a4bf12d8b174f7d83c6a0929fb78114&diffOnly=False&revisionId=ea917d18a0744b4ab411823a1b070007&doc=False) we got approval for the API changes where were the version change was included. We double checked in the e-mail whether it should be minor version bump and got feedback that from C# architects that it should be major version update as it's a behavioral breaking change in a shared library, and only got feedback from Python architects to go with a minor version bump. So we shipped with a major version update for java, js, c#

Reply:
We should ask the Java architects just to make sure as early as we can, we want to avoid having to change the version from 2.0.0 to some minor update when the time to release as GA comes.